src/CommunityToolkit.Maui/Layouts/DockLayout.shared.cs

```diff
@@ -0,0 +1,61 @@
+﻿using CommunityToolkit.Maui.Core.Interfaces;
+using CommunityToolkit.Maui.Core.Layouts;
+using Microsoft.Maui.Layouts;
+
+namespace CommunityToolkit.Maui.Layouts;
+
+/// <summary>DockLayout positions its child elements along the edges of the layout container.</summary>
+public class DockLayout : Layout, IDockLayout
+{
+	/// <summary>Docking position for a view.</summary>
+	public static readonly BindableProperty DockProperty = BindableProperty.Create(nameof(DockEnum), typeof(DockEnum), typeof(DockLayout), DockEnum.None, BindingMode.TwoWay);
```

Comment:
Why does this need to be `BindingMode.TwoWay` by default?

Reply:
I guess, we dont ...